queries.yaml

```diff
@@ -975,19 +958,77 @@ post_process:
   - fn: vectordatasource.transform.drop_properties_with_prefix
     params: {prefix: mz_}
 
+  # drop small inners in buildings before we do the merge.
+  - fn: vectordatasource.transform.drop_small_inners
+    params:
+      end_zoom: 15
+      source_layers: [buildings]
+      pixel_area: 0.25
```

Comment:
Could make this 0.5 or 1? But let's track that with a new issue and merge this as-is.